mule-extensions-api/src/main/resources/META-INF/mule-module.properties

```diff
@@ -54,6 +54,20 @@ artifact.export.classPackages=org.mule.runtime.extension.api,\
                               org.mule.runtime.extension.api.stereotype,\
                               org.mule.runtime.extension.api.util,\
                               org.mule.runtime.extension.api.property,\
+                              org.mule.sdk.api.connectivity,\
```

Comment:
you cannot export packages that don't belong to you in the first place

Reply:
My bad, These should be exported from the mule-sdk-api artifact.